lib/mix/test/mix/tasks/release_test.exs

```diff
@@ -725,6 +725,9 @@ defmodule Mix.Tasks.ReleaseTest do
         assert System.cmd(script, ["rpc", "ReleaseTest.hello_world()"], env: env) ==
                  {"hello world\n", 0}
 
+        assert System.cmd(script, ["rpc", "IO", "inspect", "foo", "bar baz", "--bat"], env: env) ==
```

Comment:
I think this will be broken:

```suggestion
        assert System.cmd(script, ["rpc", "IO", "inspect", "--foo", "bar baz", "--bat"], env: env) ==
```

:(

You will need to introduce a `--` after `mod` `fun`.

Reply:
yeah, I'm missing a atest for this.